### ISSUE TITLE
Pages Privacy Settings

### ISSUE THREAD

**Issue by SalmanKhan86** [PHPfox-Official/phpfox-v4-issues#2003]
if i enabled who can post a comment to (Members only) in Manage>Permissions than users who are not liked my page can also able to comment on my pages post.. or if i set this section to Admins only than members not able to view Pages Territory(msg;-Unable to view this section due to privacy settings.)

and if i set  Who can view/browse comments? to (Admins only) then Pages Territory does't show to anyone with msg:-Unable to view this section due to privacy settings. this settings apply only to comments or whole pages post?

**Comment by neil96** [comment]
Hi @SalmanKhan86 ,

Comment here means feed.

Regards.

**Comment by PhpFoxJohnJr** [comment]
Neil, 
Comment or Feeds it does not matter.  The part that might be the issues is this...
"if i enabled who can post a comment to (Members only)"....." if i set this section to Admins only than members not able to view Pages Territory(msg;-Unable to view this section due to privacy settings.)"

I have not check it but setting admin only for "Who can post a comment?"  should not block members seeing the post as that would be the setting "Who can view/browse comments? "

**Comment by neil96** [comment]
Hi @PhpFoxJohnJr ,

In the pages/groups permission, the text "comment" means feed. If you don't allow for members/guests view comment, it means they can't view all feeds, so they can't see middle block (feed block).

In pages, we separate setting view and post, in groups, we merge them.

Regards.

**Comment by PhpFoxJohnJr** [comment]
Ok...maybe I am losing it :)

When I look at Manage under PAGES under permissions I see the following that maybe I guess I don't understand.

Who can post a comment?
When I see the above setting....in my mind it is saying who can post a comment.  So if I put admin I would assume only the admin can post a comment.  Has nothing to do with who can see those comments.

Next setting
Who can view/browse comments?
When I see this setting I think it means what it says....who can view the comments or feed.  So if I put it to admin only then only the admin can see the comments or feed.  If I set it to members only then only members can see it.  If I set it to anyone...well you get the picture.  

Let me understand what you are saying before bringing up groups stuff.  If I am reading his issue correctly...and maybe I am not :) I thought he was saying he set the setting post comments to admin and no matter what other setting of view comments was set only the admin could see the comment.  If that is the case then why even have the view comments settings if it does not work?  But it is late and I am very tired so maybe I am thinking wrong...only going to get 4 hours of sleep tonight I guess...good night :)
